### ISSUE TITLE
admin UI for refunds

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3534]
Admin clicks a `Push Upstream` button on ~user's settings page next to their balance. We compute the prorated amounts to push back to all the credit cards it came from (ignore one-off bitcoin and PayPal pay-ins). We issue refunds in those amounts back to the credit cards.
##

**Comment by whit537** [comment]
From: Balanced

> This is a notice that we're changing our pricing for refunds on Thursday, June 11, 2015 at 11:59pm PST.
>
Previously, we returned the variable component of the debit (2.9% in standard pricing); however, we will no longer return the fees of the debit when performing a refund. We've worked hard to manage our finances during the wind-down and need to continue to do so through mid-October. This pricing change will allow us to make our costs more predictable.
